### DISCUSSION TITLE
Docker Container Unhealthy

### DISCUSSION THREAD

**Discussion by qubeley10032** [AdguardTeam/AdGuardHome#6127] [Q&A]
Hi all!
Searched around and couldn't really find an answer, so here I am..

As it says in the title, the AGH container has been marked as **unhealthy** since about a week ago. Tried restarting, clearing logs, rebuilding the container but the problem persists.
The logs has a line, "AdGuard Home updates are disabled", but I'm not sure how it could be

ANy idea what could be the issue?
Thanks

The container logs,
`2023/08/21 18:12:54.806147 [info] tls: using default ciphers

2023/08/21 18:12:54.811361 [info] safesearch default: reset 253 rules

2023/08/21 18:12:54.822135 [info] Initializing auth module: /opt/adguardhome/work/data/sessions.db

2023/08/21 18:12:54.822950 [info] auth: initialized.  users:1  sessions:3

2023/08/21 18:12:54.823116 [info] AdGuard Home updates are disabled

2023/08/21 18:12:54.823127 [info] web: initializing

2023/08/21 18:12:55.160819 [info] dnsproxy: cache: enabled, size 4096 b

2023/08/21 18:12:55.160843 [info] dnsproxy: max goroutines is set to 300

2023/08/21 18:12:55.162262 [info] clients: processing addresses

2023/08/21 18:12:55.162849 [info] AdGuard Home is available at the following addresses:

2023/08/21 18:12:55.163946 [info] go to http://127.0.0.1:3000

2023/08/21 18:12:55.163977 [info] go to http://172.18.0.2:3000

2023/08/21 18:12:59.721517 [info] dnsproxy: starting dns proxy server

2023/08/21 18:12:59.726668 [info] Ratelimit is enabled and set to 20 rps

2023/08/21 18:12:59.726715 [info] The server is configured to refuse ANY requests

2023/08/21 18:12:59.726723 [info] dnsproxy: cache: enabled, size 1000000000 b

2023/08/21 18:12:59.727386 [info] dnsproxy: max goroutines is set to 300

2023/08/21 18:12:59.727436 [info] dnsproxy: creating udp server socket 0.0.0.0:53

2023/08/21 18:12:59.727658 [info] dnsproxy: listening to udp://[::]:53

2023/08/21 18:12:59.727713 [info] dnsproxy: creating tcp server socket 0.0.0.0:53

2023/08/21 18:12:59.727830 [info] dnsproxy: listening to tcp://[::]:53

2023/08/21 18:12:59.728717 [info] dnsproxy: entering udp listener loop on [::]:53`

**Comment by CamelT0E** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/5714

**Comment by ppfeufer** [comment]
See v0.107.34 Release Notes » https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.34

> Docker HEALTHCHECK
>
> Docker users should note that the Docker HEALTHCHECK mechanism has been removed, since it was causing a lot of issues, especially when used with Podman and other popular Docker tools.
